gcp/README.md

```diff
@@ -77,3 +77,43 @@ This happens when trying to enable an API that is already enabled. This shouldn'
 ```
 rake xk"[gcloud services disable container.googleapis.com]"
 ```
+
+### Restoring CouchDB data
+
+We are considering number of probable failure scenarios for our GCP infrastructure:
+
+1. **Data corruption on a single CouchDB replica**
+
+In this scenario we rely on CouchDB ability to recover from loss of one or more replicas (our current production CouchDB settings allow us to lose up to 2 random nodes and still keep data integrity). The best course of action as follows:
```

Comment:
I'd say that is allow to lose up to (n_replicas - 1) random nodes.

Reply:
Here is [n value](http://docs.couchdb.org/en/2.1.2/cluster/theory.html) from current CouchDB settings for GCP:
```
/project # kubectl --namespace gpii exec couchdb-couchdb-0 -c couchdb -- cat /opt/couchdb/etc/default.ini | grep -E "n=[0-9]"
n=3```
Which means that we can lose exactly 2 random nodes.